woocommerce/woocommerce-ios
# [Woo Express Cleanup] Delete code related to Free Trial local notification and survey.

Closes: #12413


## Description


This PR removes code related to local notification for Woo Express site creation. One of the notifications provided a survey to the user; the survey is deleted too in this PR.

## Testing instructions


1. Since this is mostly deletion, just ensure app still builds fine and there's no test errors, as well as confidence check the deleted code. As part of the Woo Express Cleanup project, site creation is not supported anymore. Since free trial local notifications were created during site creation, the code being deleted here is not reachable by users anyway.

2. To be extra safe, you can also test in a real device with the provisioning profile to test remote notification (e.g: new order notification), to ensure that those notifications still appear. (Check paNNhX-hI-p2 for details) I have tried this and confirmed that remote notifications still appear.

PS: I'll be AFK the whole of next week so if this is needed urgently, please feel free to take over the PR.
